src/components/main/flight/FlightInfoScreen.js

```diff
@@ -239,265 +353,355 @@ export default class FlightInfoScreen extends React.Component {
-                <TouchableOpacity
-                  onPress={() => this.setState({ tripIndex: 2 })}>
-                  <Text style={styles.buttonText}>{i18n.t('ROUND TRIP')}</Text>
-                </TouchableOpacity>
+        <ImageBackground
+          source={this.background}
+          style={styles.background_image}>
+          <View style={styles.backDrop}>
+            <View style={styles.innerView}>
+              {/* Flight Information */}
+              <View style={styles.topInnerView}>
+                <View style={styles.flightView}>
+                  <Text style={styles.flightNumberLabel}>Flight Number</Text>
+                  <Text style={styles.flightNumberText}>
+                    {flightChars}
+                    {flightNums}
+                  </Text>
+                  <Text style={styles.flightInfoText}>
+                    {depCityName} ({depCityIata}) to {arrCityName} (
+                    {arrCityIata}) via {airlineName}{" "}
+                  </Text>
+                </View>
+
+                {/* Arrival Departure  Airport Names*/}
+                <View style={styles.itineraryView}>
+                  <View style={styles.itineraryViewItem}>
+                    <View>
+                      <Text style={styles.itineraryLabel}>
+                        {depAirportName}
+                      </Text>
+                    </View>
+                  </View>
+
+                  <View style={styles.plane_icon_view}></View>
+
+                  <View style={styles.itineraryViewItem}>
+                    <View>
+                      <Text style={styles.itineraryLabel}>
+                        {arrAirportName}
+                      </Text>
+                    </View>
+                  </View>
+                </View>
+
+                {/* Arrival Departure Airport Iata Codes*/}
+                <View style={styles.itineraryView}>
+                  <View style={styles.itineraryViewItem}>
+                    <Text style={styles.itineraryAirportCode}>
+                      {departureIata}
+                    </Text>
+                  </View>
+
+                  <Ionicons name={"ios-airplane"} style={styles.plane_icon} />
+
+                  <View style={styles.itineraryViewItem}>
+                    <View>
+                      <Text style={styles.itineraryAirportCode}>
+                        {arrivalIata}
+                      </Text>
+                    </View>
+                  </View>
+                </View>
+
+                {/* One Way or Two Way*/}
+                <View style={styles.seatIndexView}>
+                  <View style={styles.seatIndexViewItem}>
+                    <Text style={styles.seatIndexLabel}>One Way</Text>
+                  </View>
+
+                  <View style={styles.switchView}>
+                    <Switch
+                      trackColor={{
+                        false: COLORS.sandy,
+                        true: COLORS.sandy,
```

Comment:
Can we make the switch slightly larger?

Reply:
Switch size was adjusted to be larger and it works on IOS

Reply:
<img width="487" alt="Screen Shot 2020-07-21 at 5 44 35 PM" src="https://user-images.githubusercontent.com/22000425/88121457-984f3d80-cb7a-11ea-95cd-83ac61924851.png">

Reply:
^^^

Reply:
oof. I guess it's different in android.